README.md

```diff
@@ -233,7 +233,7 @@ The [sync-with-upstream.yaml](https://github.com/jenstroeger/python-package-temp
 
 ## Versioning, publishing and changelog
 
-To enable automation for [semantic versioning](https://semver.org/), package publishing, and changelog generation it is important to use meaningful [conventional commit messages](https://www.conventionalcommits.org/)! This package template already has a built-in semantic release support enabled which is set up to take care of all three of these aspects — every time changes are merged into the `main` branch.
+To enable automation for [semantic versioning](https://semver.org/), package publishing, and changelog generation it is important to use meaningful [conventional commit messages](https://www.conventionalcommits.org/)! This package template already has a built-in semantic release support enabled which is set up to take care of all three of these aspects — every time changes are merged into the `main` branch. Furthermore, after a new package was published, the Github Actions workflow rebases the `staging` branch automatically on top of the bumped `main` branch.
```

Comment:
Please add a short description of the current branching model first. Otherwise the additional rebasing might not make sense to users of this package.

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/536/commits/7d672a76a53f0899e0bd6c8923aeb21501d4836c.